portfolio/src/main/webapp/index.html

```diff
@@ -1,33 +1,43 @@
 <!DOCTYPE html>
 <html>
-  <head>
+<head>
     <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Matthew Wong 🎯</title>
-    <link rel="shortcut icon" type="image/jpg" href="/images/Raptors.png"/>
+    <link rel="shortcut icon" type="image/jpg" href="/images/Raptors.png" />
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
-  </head>
+</head>
 
-  <body class="cursor">
+  <body>
     <!--Top Half of Site-->
-    <div id="content">
-      <!--Introduction-->
-      <h1 class="headerFont">Matthew Wong</h1>
-      <img src="/images/ProfilePic.png" width="180" height="180" class="profilePic"/>
-      <p class="mediumFont">Swift iOS Developer • Startups 🚀</p>
-      <p class="fonts">Hello, nice to meet you! I'm a mobile developer from Toronto, Canada. I'm currently working @ theScore as an iOS Developer, and previously worked @ Shopify in a technical support/ecommerce consulting role!</p>
-
-      <!--Social Media Icons-->
-      <a href="https://github.com/MattWong-ca"><img src="/images/GitHub.png" class="socialMediaIcons"/></a>
-      <a href="https://www.linkedin.com/in/mattwong-ca/"><img src="/images/LinkedIn.png" class="socialMediaIcons"/></a>
-      <a href="https://twitter.com/mattwong_ca"><img src="/images/Twitter.png" class="socialMediaIcons"/></a>
-      <a href="https://www.instagram.com/mattwong.ca/"><img src="/images/Instagram.png" class="socialMediaIcons"/></a>
-      <a href="https://www.f6s.com/mattwong.ca"><img src="/images/F6S.png" class="socialMediaIcons"/></a>
+    <div class="landingPage">
+        <!--Introduction-->
+        <h1 class="nameTitleFont">Matthew Wong</h1>
+        <img src="/images/ProfilePic.png" width="230" height="230" class="profilePic" />
+        </br>
 
+        <!-- Typing effect -->
+        <a class="typewrite hi mediumFont" data-period="2000"
+            data-type='["Swift iOS Developer", "Java Android Developer", "Aspiring Founder"]'>
+            <span class="wrap">|</span>
+        </a>
+
+        <p class="fonts">
+            Hello, nice to meet you! I'm a mobile developer from Toronto, Canada. I'm currently working @ theScore
+            as an iOS Developer, and previously worked @ Shopify in a technical support/ecommerce consulting role!
```

Comment:
optional: make the company name text a link to their page. I know you do this with your experience section below, but it's not so obvious that those app icons are the same companies as here.

Not required.

Reply:
Added!